### ISSUE TITLE
Report Results broken in Jenkins build of 3.1.1 (Trac #486)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#620]
Windows 7.
Spotted by Richard, verified by Steve.

Report Results is broken in the Jenkins build of 3.1.1. Works in the release version of 3.1.1 and 3.0.0.

We initially thought it was a feature of the recent forced update of Adobe Acrobat Reader from X to DC, but DC works with the earlier versions. 

Migrated from http://trac.sasview.org/ticket/486
```json
{
    "status": "closed",
    "changetime": "2015-11-28T23:18:36",
    "_ts": "2015-11-28 23:18:36.165535+00:00",
    "description": "Windows 7.\nSpotted by Richard, verified by Steve.\n\nReport Results is broken in the Jenkins build of 3.1.1. Works in the release version of 3.1.1 and 3.0.0.\n\nWe initially thought it was a feature of the recent forced update of Adobe Acrobat Reader from X to DC, but DC works with the earlier versions. ",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2015-11-26T17:56:11",
    "component": "SasView",
    "summary": "Report Results broken in Jenkins build of 3.1.1",
    "priority": "blocker",
    "keywords": "",
    "milestone": "SasView 3.1.2",
    "owner": "butler",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2015/11/26 18:16:09`: 
* **butler** commented:

Seems like this should be fixed before we release 3.1.2 if it was working in 3.1.1?  Should probalby check on local builds first to see if a Jenkins build problem or a problem introduced by wxID fixes for example.


* **butler** changed milestone from "SasView Next Release +1" to "SasView 3.2.2"

**Comment by butlerpd** [comment]
Trac update at `2015/11/28 23:11:55`: 
* **butler** commented:

Looks like problem was from wxID cleanup where all -1 were replaced with a wx.ID_ANY which is much better for searching later but requires an import wx statement which one of the files, pagestate, did not have.  Checking all other files changed to make sure that was the only such instances.


* **butler** changed owner from "" to "butler"
* **butler** changed status from "new" to "accepted"
